.github/workflows/common-docker.yml

```diff
@@ -208,6 +212,13 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Login to Chainguard Container Registry
```

Comment:
maybe we should make this step optional if both CGR_USERNAME & CGR_PASSWORD are not empty otherwise it means every workflows will run with chainguard